src/components/WorkflowGraph/utils.ts

```diff
@@ -116,3 +116,33 @@ export const getTaskTypeFromCompiledNode = (
     }
     return null;
 };
+
+export const getNodeNameFromDag = (dagData: dNode, nodeId: string) => {
+    const id = nodeId.slice(nodeId.lastIndexOf('-') + 1);
+    const value = dagData[id].value;
+
+    if (value.taskNode) {
+        return value.taskNode.referenceId.name;
+    } else if (value.workflowNode) {
+        const {
+            workflowNode: {
+                subWorkflowRef: { name }
+            }
+        } = value;
+        return name;
+    }
+    return '';
+};
+
+export const transformWorkflowToKeyedDag = (workflow: Workflow) => {
+    if (!workflow.closure?.compiledWorkflow) return {};
+
+    const dagData = transformerWorkflowToDAG(
```

Comment:
nit: Dag (sentence case - similar to other places, where you used Dag in function name)

Reply:
Pushed the fix.